src/ExampleTable.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { FunctionComponent } from 'react';
+import { Table, Divider, Button } from "antd";
+import { ColumnProps } from "antd/lib/table/";
+import { RandomData } from "./ExampleSearch"
+
+interface IRandomData {
+  data: RandomData[],
+  onEdit: (editData: RandomData) => void
+  onDelete: (deleteData: RandomData) => void
+}
+
+export const ExampleTable: FunctionComponent<IRandomData> = (data) => {
```

Comment:
💡 TIL: didn't know there's a `FunctionComponent`, seems to be the same thing as `React.FC`.